tool/build.dart

```diff
@@ -319,18 +363,20 @@ _main(List<String> argv) async {
       ], writeToBuffer: true);
       if (pur.exitCode != 0) {
         await fail(
-          "DSLink ${name}: Failed to build the DSLink.\n${pur.output}");
+          "[${name}] Failed to build the DSLink.\n${pur.output}",
+          firstLineOnlyDebug: true
```

Comment:
firstLineOnlyDebug is a specifier for when command output is provided. In debug (with `--debug`) mode, this would be duplicated.